src/bin/wasm-tools/dump.rs

```diff
@@ -853,7 +853,10 @@ macro_rules! define_visit_operator {
                         "{}"
                         $( $(, " ", stringify!($arg), ":{:?}")* )?
                     ),
-                    stringify!($visit).strip_prefix("visit_").unwrap(),
+                    match stringify!($visit).strip_prefix("visit_").unwrap() {
+			"typed_select_multi" => "typed_select",
+			other => other,
+		    },
```

Comment:
Since this is just a debugging utility I think it's reasonable to print `typed_select_multi` in this case

Reply:
Done.